main.go

```diff
@@ -39,14 +56,16 @@ func main() {
 		log.Fatalf("Error setting MySQL logger: %v", err)
 	}
 
-	// Check if "version" arg is passed.
-	argsWithProg := os.Args
+	args := flag.Args()
 
-	if len(argsWithProg) > 1 {
-		switch argsWithProg[1] {
-		case "version":
-			fmt.Println("LazySQL version: ", version)
-			os.Exit(0)
+	if len(args) > 1 {
+		fmt.Fprintln(os.Stderr, "Only a single connection is allowed")
+		os.Exit(1)
+	} else if len(args) == 1 {
+		err := components.InitFromArg(args[0])
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
 		}
 	}
```

Comment:
I would like to suggest something like this.

```suggestion
	switch len(args) {
		case 0:
			// nothing to do
    case 1:
			err := components.InitFromArg(args[0])
			if err != nil {
				fmt.Fprintln(os.Stderr, err)
				os.Exit(1)
 			}
		default:
			fmt.Fprintln(os.Stderr, "Only a single connection is allowed")
			os.Exit(1)
 	}
```



The reason is that testing the same variable in multiple if statement makes it a perfect candidate to use a switch

Reply:
Good practice! Updated and I'll keep it in mind for future changes :))